tests/e2e/backup_restore_suite_test.go

```diff
@@ -209,12 +284,12 @@ var _ = Describe("AWS backup restore tests", func() {
 			// Wait for namespace to be deleted
 			Eventually(IsNamespaceDeleted(brCase.ApplicationNamespace), timeoutMultiplier*time.Minute*2, time.Second*5).Should(BeTrue())
 
-			updateLastInstallingNamespace(brCase.ApplicationNamespace)
+			updateLastInstallTime()
 			// run restore
 			log.Printf("Creating restore %s for case %s", restoreName, brCase.Name)
 			restore, err := CreateRestoreFromBackup(dpaCR.Client, namespace, backupName, restoreName)
 			Expect(err).ToNot(HaveOccurred())
-			Eventually(IsRestoreDone(dpaCR.Client, namespace, restoreName), timeoutMultiplier*time.Minute*4, time.Second*10).Should(BeTrue())
+			Eventually(IsRestoreDone(dpaCR.Client, namespace, restoreName), timeoutMultiplier*time.Minute*60, time.Second*10).Should(BeTrue())
```

Comment:
Same here for VSR

Reply:
Actually VSB/VSR is already checked on failure